src/clj/game/core/flags.clj

```diff
@@ -433,3 +433,37 @@
              (or (is-type? card "Operation") (rezzed? card)))
         (and (in-discard? card) (:seen card))
         (#{:scored :current} (last zone)))))
+
+(defn ab-can-prevent?
+  "Checks if the specified ability definition should prevent.
+  Checks for a :req in the :prevent map of the card-def.
+  Defaults to false if req check not met"
+  ([state side card req-fn]
+   (ab-can-prevent? state side (make-eid state) card req-fn))
+  ([state side eid card req-fn]
+   (cond
+     req-fn (req-fn state side eid card nil)
+     :else false)))
```

Comment:
Any reason we want this to default to `false`? For abilities, a missing `:req` defaults to `true`. Then we don't have to put `:req (req true)` everywhere.

Reply:
There was a rationale.  Honest....

Maybe... I think it was that if default to false means we don't stick warnings on the UI if someone forgets to write a req so we default to OFF... with abilities we default to True/ON as they have been played.  Protection is something to be turned ON not something to default ON?

It seemed right at the time - but fine to change it if folks want.  I think we have other stuff where we always want (req true) like :silent and :interactive - so we are nothing if not inconsistent :D

Reply:
I didn't think about `:silent` or `:interactive`. Guess it does depend on the situation. I don't know that I have a preference given the different use cases.